code/tracing/categories.h

```diff
@@ -0,0 +1,92 @@
+

```

Comment:
A doxy'd comment block in this file, or in the tracing.h file, should be made to outline maintenance of the tracing system and what to do if devs want to add new trace points.